src/app/content/styles/constants.ts

```diff
@@ -1,4 +1,4 @@
-import { headerHeight, topBottomMargin } from '../HighlightStyles';
```

Comment:
It is good to call to move this file, however, there is the `filter` constant in it which is used only in filters so I would recommend splitting these file to two separate files

Reply:
Filter stuff will eventually be needed for this modal as well.

Reply:
You are correct!